lib/algo/DMD.C

```diff
@@ -438,14 +438,14 @@ DMD::createSnapshotMatrix(std::vector<Vector*> snapshots)
 }
 
 void
-DMD::summary(std::string output_path)
+DMD::summary(std::string output_path, int window)
```

Comment:
Would it be better to put  "/window" + std::to_string(window) in the output_path and keep this function the same as it was before, since some people may only use regular DMD and not time-windowing DMD?

Reply:
Thanks for the suggestion, Kevin. I will change that.